MWC 4K 2021 wiki article fixes (#5795)
ppy/osu-wiki: wiki/Tournaments/MWC/2021_4K/en.md

1. The osu!mania 4K World Cup is a country-based team tournament, played on the osu!mania game mode.
   - *Notice: While this competition has been planned as a 3 versus 3 setup, this might change depending on the amount of registrations.*
2. Beatmap scoring will be based on Score V2.
3. The beatmaps for each round will be announced by the Mappool Selector Team on the Sunday before the actual matches take place.

>    - Each mappool, with the exception of the Qualifier stage, will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. 4-4 in a best of nine.

Reviewer: ```suggestion
   - Each mappool, with the exception of the Qualifier stage, will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. 4-4 in a best of 9.
```

   - If a rematch happens, the original roster for each team during that particular beatmap must remain the same. If that is not possible, e.g. by virtue of a technical issue, *both* teams will be allowed to swap rosters.
   - This rule is not to be abused. Referees may veto a rematch request if they find that this is the case.
10. Beatmaps must not be reused in the same match, with the exception of rematches.
11. **The maximum size for a team is 6 players, and the minimum is 3.**
    - This might change depending on the amount of registrations.
12. If less than the minimum amount of required players are present at match time, it can be postponed for up to 10 minutes. If, after this period, there are still not enough players for either, a *win by default* will be declared for the team with the most members present.
    - The minimum amount of required players is defined as the amount of players needed to play a beatmap.
13. Exchanging players during games is allowed without limitations.
14. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalities being applied.

> 15. If a player disconnects between beatmaps and the team cannot provide a substitute, the match can be delayed for up to 10 minutes (limited to once per team, per match). This timer is di

Reviewer: > This timer is di

?

16. All players and staff must be treated with respect. Instructions of the referees and the Tournament Management are to be followed. Decisions labeled as final are not to be objected.
17. Disrupting the match by foul play, insulting and provoking other players or staff, delaying the match or other deliberate inappropriate misbehaviour are strictly prohibited.
18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
    - Breaking the chat rules results in a silence. Silenced players cannot participate at multiplayer matches and must be exchanged for the time being.
19. Unexpected incidences will be handled by the Tournament Management. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
20. Penalties for violating the tournament rules include, but are not limited to:
    - Exclusion of specific players for one beatmap.
    - Exclusion of specific players for an entire match.

>     - Declaring the match as forfeited or won by default (wbd).

Reviewer: this sentence did make sense tbh as it was, but this works too.

```suggestion
    - Declaring the match as forfeited, or as a win by default for the other team.
```

   - Captains are expected to choose team members with honesty and good will, with the aim of fielding the strongest team possible.
   - Captains are allowed to exclude themselves from the team list.
   - Captains are allowed to conduct and coordinate "tryouts", tests to gauge player aptitude, on their own terms.
   - Captains may choose, with the approval of the Tournament Management, to delegate their role to other prospective team members.
3. To ensure valid and serious registrations, every registered user will be manually checked by the Tournament Management.
   - Every registered user will be assigned to their respective country's candidate list.
   - To be successfully accepted on the list, players are required to be placed #5000 and up on the global osu!mania 4K performance ranking, and have not violated the [osu! community rules](/wiki/Rules) within the last 12 months.
4. All successfully formed teams will be published after the Registration Phase.
   - A team should have at least 3 players registered to have the chance to play the Qualifier stage.
   - The top 32 seeded teams will qualify to the Round of 32.

> 5. Organisers, mappool selectors and referees must not participate as a player in the tournament.

Reviewer: ```suggestion
5. Organisers, Mappool Selectors and referees must not participate as a player in the tournament.
```

> ### Qualifier instructions

Reviewer 2: As mentioned in the previous comment, please confirm if map style distributions will be a thing or not.

Reviewer 2: If it takes time to confirm though let me know so we can update the rest at least.

Author: It's going to take a while to confirm it and there's some info that needs to be updated ASAP. I think it's fine if we leave this for later (probably when the pool is released).
